bastion.tf

```diff
@@ -20,13 +20,13 @@ data "aws_ami" "bastion-ami" {
 resource "aws_instance" "bastion" {
```

Comment:
Just "bastion"?  Amend to Environment name + bastion?

Reply:
Same as above, bastion is just a generic resource name, the Name tag uses the environment variable and this will be the name of the resource in aws